### PR TITLE
Create initial spec for mapml2geojson.

### PR DESCRIPTION
**Pull request by prushforth**
- [x] mapml2geojson

Will add geojson2mapml in future PR

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -860,6 +867,69 @@ <h4>The <code>&lt;<dfn id="the-layer-element">layer</dfn>&gt;</code> element</h4
...
+          function.  If the input <a href="#the-properties-element"><code>properties</code></a>
+          element does not conform to that structure, incorrect output MAY result.
+        </p>
+        <p>
+          The <a href="#dom-geojsongenerationoptions-transform"><code>transform</code></a> option, 
+          allows the author to trigger the reverse projection transformation of 
+          the coordinates of feature geometries when converting to the GeoJSON 
+          [[rfc7946]] format. By default 
+          (<a href="#dom-geojsongenerationoptions-transform"><code>transform</code></a> === <code>false</code>),
+          no reverse projection transformation of coordinates will be performed.
```
Comment (AliyanH) [inline spec/index.html]: With the current implementation, the transform option is set to 'true' by default, so it tries to do the reverse projection transformation of the coordinates, unless explicitly told not to through the transform=false option.

Should the implementation be changed (minor fix)?

File: spec/index.html [hunk 2]
```diff
@@ -786,11 +786,10 @@ <h4>The <code>&lt;<dfn id="the-layer-element">layer</dfn>&gt;</code> element</h4
...
         <dt><a href="https://html.spec.whatwg.org/multipage/dom.html#concept-element-attributes">Content attributes</a>:</dt>
         <dd><a href="https://html.spec.whatwg.org/multipage/dom.html#global-attributes">Global attributes</a>.</dd>
-        <dd id="attr-layer-src"><code>src</code> — Address of the resource.</dd>
-        <dd id="attr-layer-label"><code>label</code> — String label for the layer in the layer control, if displayed.</dd>
-        <dd id="attr-layer-checked"><code>checked</code> — Layer on/off status.</dd>
-        <dd id="attr-layer-disabled"><code>disabled</code> — Sets the disabled state of the layer in the layer control.</dd>
-        <dd id="attr-layer-hidden"><code>hidden</code> — Visibility status of the layer in the layer control.</dd>
+        <dd id="attr-layer-src"><code>src</code> — The URL of the layer resource.</dd>
+        <dd id="attr-layer-label"><code>label</code> — A label for the layer in the layer control, if the layer is not not <code>hidden</code>. The map author's <code>title</code> element takes precedence over <code>label</code>, if available.</dd>
```
Comment (AliyanH) [inline spec/index.html]: Add a link to the title element?

File: spec/index.html (hunk 1 above)
Comment (prushforth) [reply spec/index.html]: I changed the description.  That should teach me to ad lib late in the afternoon.  Thanks for catching this!

File: spec/index.html (hunk 2 above)
Comment (prushforth) [reply spec/index.html]: Done

**Review by AliyanH — APPROVED** [review]

**Pull request merged**